src/engraving/dom/slurtie.cpp

```diff
@@ -57,6 +57,43 @@ SlurTieSegment::SlurTieSegment(const SlurTieSegment& b)
     mutldata()->path.set_value(b.ldata()->path());
 }
 
+bool SlurTieSegment::isEditAllowed(EditData& ed) const
+{
+    if (ed.key == Key_Home && !ed.modifiers && ed.hasCurrentGrip()) {
+        return true;
+    }
+
+    return false;
+}
+
+//---------------------------------------------------------
+//   edit
+//    return true if event is accepted
+//---------------------------------------------------------
+
+bool SlurTieSegment::edit(EditData& ed)
+{
+    if (!isEditAllowed(ed)) {
+        return false;
+    }
+
+    if (ed.key == Key_Home && !ed.modifiers) {
+        if (ed.hasCurrentGrip()) {
+            startEditDrag(ed);
```

Comment:
Looks like it isn't necessary, usually NotationInteraction does it before editing